src/servers/src/http/table_result.rs

```diff
@@ -143,7 +142,7 @@ impl IntoResponse for TableResponse {
             .into_response();
         resp.headers_mut().insert(
             &GREPTIME_DB_HEADER_FORMAT,
-            HeaderValue::from_static("TABLE"),
+            HeaderValue::from_static(ResponseFormat::Table.as_str()),
```

Comment:
Case changed for TABLE, CSV, ARROW (now in lowercase). But I suppose it's OK and we use a constant rather than some literal string that is unassociated brittle.

Reply:
`greptimedb_v1` and `influxdb_v1` retains so it won't break cloud setting (I hope). Other formats are added recently and for human readable mostly. I suppose they are not integrated in other systems.